lib/texvc.sty

```diff
@@ -0,0 +1,117 @@
+%%
+%% This is file `texvc.sty',
+%% generated with the docstrip utility.
+%%
+%% The original source files were:
+%%
+%% texvc.dtx  (with options: `package')
+%% This is a generated file.
+%% Copyright (C) 2015 by Moritz Schubotz
+%% This file may be distributed and/or modified under the
+%% conditions of the LaTeX Project Public License, either
+%% version 1.3 of this license or (at your option) any later
+%% version. The latest version of this license is in:
+%% http://www.latex-project.org/lppl.txt
+%% and version 1.3 or later is part of all distributions of
+%% LaTeX version 2005/12/01 or later.
+\NeedsTeXFormat{LaTeX2e}[2005/12/01]
+\ProvidesPackage{texvc}
+[2015/11/09 v1.0 Initial Version]
+\RequirePackage{amsmath}
+\RequirePackage{amsfonts}
+\RequirePackage{amssymb}
+
+\PassOptionsToPackage{dvips,usenames}{color}
+\RequirePackage{color}
+
+%%<package>\PassOptionsToPackage{greek}{babel}
+%%<package>\RequirePackage{babel}
+%%<package>\RequirePackage{teubner}
+
+\RequirePackage{eurosym}
+
+\RequirePackage{cancel}
+\newcommand{\darr}{\downarrow}
+\newcommand{\dArr}{\Downarrow}
+\newcommand{\Darr}{\Downarrow}
+\newcommand{\lang}{\langle}
+\newcommand{\rang}{\rangle}
+\newcommand{\uarr}{\uparrow}
+\newcommand{\uArr}{\Uparrow}
+\newcommand{\Uarr}{\Uparrow}
+\newcommand{\C}{\mathbb{C}}
+\renewcommand{\H}{\mathbb{H}}
+\newcommand{\N}{\mathbb{N}}
+\newcommand{\Q}{\mathbb{Q}}
+\newcommand{\R}{\mathbb{R}}
+\newcommand{\Z}{\mathbb{Z}}
+\newcommand{\alef}{\aleph}
+\newcommand{\alefsym}{\aleph}
+\newcommand{\Alpha}{\mathrm{A}}
+\renewcommand{\and}{\land}
+\newcommand{\ang}{\angle}
+\newcommand{\Beta}{\mathrm{B}}
+\newcommand{\bull}{\bullet}
+\newcommand{\Chi}{\mathrm{X}}
+\newcommand{\clubs}{\clubsuit}
+\newcommand{\cnums}{\mathbb{C}}
+\newcommand{\Complex}{\mathbb{C}}
+\newcommand{\Dagger}{\ddagger}
+\newcommand{\diamonds}{\diamondsuit}
+\renewcommand{\Doteq}{\doteqdot}
+\renewcommand{\doublecap}{\Cap}
+\renewcommand{\doublecup}{\Cup}
+\renewcommand{\empty}{\emptyset}
+\newcommand{\Epsilon}{\mathrm{E}}
+\newcommand{\Eta}{\mathrm{H}}
+\newcommand{\exist}{\exists}
+\renewcommand{\ge}{\geq}
+\renewcommand{\gggtr}{\ggg}
+\newcommand{\hAar}{\Leftrightarrow}
+\newcommand{\harr}{\leftrightarrow}
+\newcommand{\Harr}{\Leftrightarrow}
+\newcommand{\hearts}{\heartsuit}
+\newcommand{\image}{\Im}
+\newcommand{\infin}{\infty}
+\newcommand{\Iota}{\mathrm{I}}
+\newcommand{\isin}{\in}
+\newcommand{\Kappa}{\mathrm{K}}
+\newcommand{\larr}{\leftarrow}
+\newcommand{\Larr}{\Leftarrow}
+\newcommand{\lArr}{\Leftarrow}
+\renewcommand{\le}{\leq}
+\newcommand{\lrarr}{\leftrightarrow}
+\newcommand{\Lrarr}{\Leftrightarrow}
+\newcommand{\lrArr}{\Leftrightarrow}
+\newcommand{\Mu}{\mathrm{M}}
+\newcommand{\natnums}{\mathbb{N}}
+\renewcommand{\ne}{\neq}
+\newcommand{\Nu}{\mathrm{N}}
+\renewcommand{\O}{\emptyset}
+\newcommand{\omicron}{\mathrm{o}}
+\newcommand{\Omicron}{\mathrm{O}}
+\newcommand{\or}{\lor}
```

Comment:
The problem is that \or does not compile, with TeX. Thus we need to develop a workaround for that.